docs/eureka/integration-guide.mdx

```diff
@@ -58,7 +58,13 @@ If you're brand new to Skip Go, read about our cross-chain developer platform on
 
 ### Current Skip Go Integrator
 
-Ensuring Eureka works with your Skip Go integration is the same easy process as any other bridge! Changes are as follows:
+Ensuring Eureka works with your Skip Go integration is the same easy process as any other bridge!
+
+<Note>
+To get routes involving Eureka assets, you must include `"experimental_features": ["eureka"]` in your `/route` and `/msgs_direct` payload. This requirement applies to all routes involving Eureka assets, including same-chain swaps. Without this flag, Eureka assets will not be included in routing responses.
```

Comment:
I would change this to clarify you need to add "eureka" to the experimental_features array being passed in. Which looks like what you have written if they don't have any other experimental features turned on, but can also look like "eureka" added to other items in the array if they have others turned on as well.